src/neptune/internal/backends/neptune_backend.py

```diff
@@ -303,7 +304,7 @@ def search_leaderboard_entries(
         types: Optional[List[ContainerType]] = None,
         query: Optional[NQLQuery] = None,
         columns: Optional[List[str]] = None,
-    ) -> List[LeaderboardEntry]:
+    ) -> Generator[LeaderboardEntry, None, None]:
```

Comment:
Do we have to change anything in non-hosted implementations of this? Mock/Offline one?